Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -11899,7 +11906,9 @@ def send_data_to_xsiam(data, vendor, product, data_format=None, url_key='url', n
         'collector-type': ASSETS if data_type == ASSETS else EVENTS
     }
     if data_type == ASSETS:
-        headers['snapshot-id'] = str(round(time.time() * 1000))
+        if not snapshot_id:
```

Comment:
let's add test case for both cases

Reply:
added test "test_send_data_to_xsiam_custom_snapshot_id_and_items_count"